fusestore/store/src/api/rpc/flight_service_test.rs

```diff
@@ -2,13 +2,16 @@
 //
 // SPDX-License-Identifier: Apache-2.0.
 
+use common_arrow::arrow::array::ArrayRef;
+use common_datablocks::DataBlock;
 use common_flights::GetTableActionResult;
+use common_flights::StoreClient;
 use log::info;
 use pretty_assertions::assert_eq;
+use test_env_log::test;
 
-#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+#[test(tokio::test)]
```

Comment:
`#[tokio::test(flavor = "multi_thread", worker_threads = 1)]`

Why is the thread hint is removed? 🤔 Does it affect the tests?

Reply:
To avoid conflicts of env logger initialization between test cases,  or tedious check like `env_logger::builder().is_test(true).try_init()`,  [test_env_log](https://crates.io/crates/test-env-log) is used.

Unfortunately,  `test_env_log::test` only supports a simple form of attribute declaration( If I get it right),  things like `#[test(tokio::test(flavor... ))]` does not work ;(

Till now, unit tests work pretty well both locally and in CI env.

Reply:
I see...
Thanks for the crystal clear explanations!